blog/_posts/2023-01-31-scala-center-2023-roadmap.md

```diff
@@ -0,0 +1,259 @@
+while the debugger is paused in the middle of a program. Step-by-step execution now automatically skips the methods
+generated by the compiler (e.g. mix-in forwarders, getters, setters, bridge methods, and synthetic methods of case
+classes). Learn more in [this video](https://www.youtube.com/watch?v=UePrOa_1Am8).
+
+**We fixed the apps installed by the [setup instructions](https://docs.scala-lang.org/getting-started).** The
+command-line apps `sbt` and `scala` now behave consistently regardless of the installation process (`cs install` vs
+manual download of the release artifacts).
+
+### Community and Contributor Experience
+
+**We co-organized the [ScalaCon](https://www.scalacon.org/) online conference and participated in several in-person community
+events.** In April 2022, for the first time in 2.5 years, events could finally take place in person. Our team encouraged
+many Scala User Groups to restart their regular Scala Meetups. We gave talks and organized [Open-Source Sprees](https://github.com/scalacenter/sprees)
+at local events in Europe. We were also involved in the online conference ScalaCon: we were in charge of the program and
+the program committee, and we presented several talks including a keynote [Towards a Healthy & Resilient Scala Community](https://youtu.be/svWnwU5PXxE)
+where we presented our strategy for the governance of Scala. The talk videos of the whole conference are available online
+[here](https://www.youtube.com/playlist?list=PL3t6y1EktOPhflaXB28Kw7kwRb30K0W-P).
+
+**We published videos about Scala 3.** [Let’s Talk About Scala 3](https://www.youtube.com/playlist?list=PLTx-VKTe8yLxYQfX_eGHCxaTuWvvG28Ml)
+is a series of videos where we share interesting, useful, and cool things related to Scala 3. We published 4 new
+videos in 2022.
+
+**We improved the code editing experience in [Scastie](https://scastie.scala-lang.org).** Scastie is the online Scala
+code editor that is used every day by one thousand developers to share code snippets in the community. We have 
+implemented “IDE features” such as autocompletion and showing type information under the mouse pointer. Under the hood,
+we reused some components of Metals.
+
+**We mentored new contributors to the Scala 3 compiler.** The Compiler
+Academy organizes online pair-programming events to mentor new contributors on the compiler codebase. You can learn
+more about the Compiler Academy in [this blog article]({% link blog/_posts/2022-11-02-compiler-academy.md %}).
+
+**We implemented the building blocks of a new tool to detect incompatibilities between programs.** Guaranteeing the
+absence of incompatibilities between library dependencies is the cornerstone of a seamless Scala ecosystem. We have
+implemented a prototype of a new tool to detect incompatibilities between two versions of a program (like
+[MiMa](https://github.com/lightbend/mima), but based on the TASTy representation of programs instead of JVM bytecode).
```

Comment:
Just a note. It might be a good idea here to expand on _why_ we did this. For example `MiMa` already existed, so how does this benefit the community? I think it's important to illustrate this to make it not seem like we just built a tool no one asked for and in place of one that many already use.

Reply:
Good point! I’ve added a link to a document that explains that in 79c0d585